src/Chapter2.hs

```diff
@@ -373,7 +391,7 @@ not False = True
 
 To perform pattern-matching, repeat a function name as many times as
 many patterns you want to cover. The cool thing about Haskell is that
-the compiler warns you if you forget to cover some cases. So you
+the compiler warns you if you forget to cover some cases (Question: There is a way to bring the warning into an error?). So you
```

Comment:
Yes, there is the compiler option you can pass to get errors on all warnings: [`-Werror`](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/using-warnings.html#ghc-flag--Werror)